client/providers/OmnichannelProvider.tsx

```diff
@@ -106,6 +107,19 @@ const OmnichannelProvider: FC = ({ children }) => {
 					limit: omnichannelPoolMaxIncoming,
 				},
 			).fetch();
+
+			return inquires.map((inquire) => {
+				const { source } = Rooms.findOne(
```

Comment:
When you aren't in a room this error is thrown
<img width="683" alt="Screen Shot 2021-09-27 at 9 51 53 AM" src="https://user-images.githubusercontent.com/20868078/134911555-1c56297d-a057-463d-8d71-1b424cc6ebb0.png">